tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDtdData.java

```diff
@@ -269,7 +273,7 @@ private void checkEmpty(Multimap<String, String> m, DtdType type, Element elemen
                     logKnownIssue("cldrbug:9982", "Lower priority fixes to bad xml");
                     break;
                 default:
-                    m.put("error", "\t||" + showPath(parents) + "||path has both children AND value attributes"
+                    m.put("error", "\t||" + showPath(parents) + "||DTD has both children AND value attributes: tr35.md#XML_Format"
```

Comment:
This line is already _very long_ in practice, and will (should) only be hit by poor souls trying to update the DTD, so did not include the full URL.